xbmc/music/Song.cpp

```diff
@@ -165,8 +165,8 @@ const std::vector<std::string> CSong::GetArtist() const
   //When artist credits have not been populated attempt to build an artist vector from the descrpition string
   //This is a tempory fix, in the longer term other areas should query the song_artist table and populate
   //artist credits. Note that splitting the string may not give the same artists as held in the song_artist table
-  if (songartists.empty() && !strArtistDesc.empty())
-    songartists = StringUtils::Split(strArtistDesc, g_advancedSettings.m_musicItemSeparator);
+  //if (songartists.empty() && !strArtistDesc.empty())
+  //  songartists = StringUtils::Split(strArtistDesc, g_advancedSettings.m_musicItemSeparator);
```

Comment:
Not something I would normally do, and I would expect to remove before merge. There is just a doubt in my mind that some other part of Kodi is using the description string to  make artists. By commenting out it will show up in testing. I really want this built so others can test it, and @Montellese to have some chance to look at it.  Not suggesting we merge before that has happened.